<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>ResultDryLab</title>
    <link rel="stylesheet"
        href="https://2017.igem.org/wiki/index.php?title= Template:SCU_China/default_CSS &action=raw&ctype=text/css">
    <link rel="stylesheet" href="css/reset.css">
    <link rel="stylesheet" href="css/common.css">
    <link rel="stylesheet" href="css/contentCommon.css">
    <style>
        #header #title {
            left: 15%;
        }

        #contain .left img {
            position: absolute;
            width: 70%;
            top: 180px;
            left: 20%;
            z-index: 1;
        }
    </style>
</head>

<body>
    <header id="header">
        <div id="load" v-show="loading"><img src="./data/T--CPU_CHINA--loading.gif" alt=""></div>
        <div id="menu">
            <div id="menuBack" v-show="isOpen" :class="isActive" @click="ifclick"></div>
            <a href="../index.html"><img id="logo" src="./data/T--CPU_CHINA--ProjectLogo.png" alt="项目logo"></a>
            <div id="select" @mouseleave="leave" @mouseover="on">
                <img id="menuImg" :src="menu" alt="这是菜单图片" @mouseover="mouseover" @click="click">
                <ul id="class-a" class="class" v-show="classA">
                    <li v-for="(item,index) in classList" :class="isActiveA == index? 'class-a' : ''"
                        @click="classAClick(index)">{{item}}</li>
                </ul>
                <ul id="class-b" class="class" v-show="classB">
                    <li v-for="item in classList2[isActiveA]" :class="isActiveB == item.num? 'class-b' : ''"
                        @mouseover="classBOver(item.num)" @mouseleave="classBLeave(item.num)"><a :href="item.url">
                            {{item.text}}</a></li>
                </ul>
            </div>
        </div>
        <img class="background" src="./data/TopBackground/3 result/T--CPU_CHINA--DryLab--Background.png" alt="">
        <h1 id="title">RESULT DRY LAB</h1>
        <svg width="100%" height="100%" viewBox="0 0 1920 600" version="1.1"
            style="fill-rule:evenodd;clip-rule:evenodd;stroke-linejoin:round;stroke-miterlimit:2;">
            <g transform="matrix(1.16285,0,0,1.38837,253.426,-231.748)">
                <g id="AD1">
                    <path id="凹形遮罩"
                        d="M1433.18,439.801C1433.18,439.801 1001.08,524.071 607.621,524.374C219.99,524.673 -217.935,439.801 -217.935,439.801L-217.935,944.811L1433.18,944.811L1433.18,439.801Z"
                        style="fill:#FAFAFA;" />
                </g>
            </g>
        </svg>
    </header>
    <div id="contain" class=" clearfix">
        <div class="left clearfix">
            <svg class="Frosting" width="70%" height="280px">
                <defs>
                    <pattern id="cardBg" x="0" y="0" width="100" height="100" patternUnits="userSpaceOnUse">
                        <image opacity="0.5" width="100" height="100" xlink:href="./data/T--CPU_CHINA--Frosting1.png" />
                        <image class="fillLight" opacity="0.5" width="100" height="100"
                            xlink:href="./data/T--CPU_CHINA--Frosting2.png" />
                    </pattern>
                </defs>
                <g class="main">
                    <g class="card">
                        <rect fill="none" stroke="#ccc" opacity="0.75" stroke-width="3" width="100%" height="100%"
                            rx="10" ry="10" opacity="0.6" />
                        <rect fill="url(#cardBg)" width="100%" height="100%" rx="11" ry="11" opacity="0.6" />
                    </g>
                </g>
            </svg>
            <nav>
                <div class="catalogue-title">RESULT DRY LAB</div>
                <ul class="catalogue">
                    <li class="num">
                        <a href="#section1" title="MOLECULAR DYNAMIC SIMULATIONS ANALYSIS">MOLECULAR DYNAMIC SIMULATIONS
                            ANALYSIS</a>
                    </li>

                    <li class="num">
                        <a href="#section2" title="OPTIMIZATION OF MUTANT LIBRARY">OPTIMIZATION OF MUTANT LIBRARY</a>
                    </li>

                    <li class="num">
                        <a href="#section3" title="REFERENCE">REFERENCE</a>
                    </li>
                </ul>
            </nav>
            <img src="./data/Left/T--CPU_CHINA--Keyboarding.png" alt="">
        </div>
        <div id="detail" class="clearfix">
            <div class="section" id="section1">
                <h2 id='molecular-dynamic-simulations-analysis'><span>MOLECULAR DYNAMIC SIMULATIONS ANALYSIS</span></h2>
                <p><span>To discover the innate structural properties of manganese peroxidase (MnP), we operated
                        molecular dynamic
                        simulation analysis on Gromacs.</span></p>
                <h3> RMSD and RMSF analysis</h3>
                <p><span>First, though the analysis of the track file of MD simulation, RMSD</span>
                    <mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg
                            xmlns="http://www.w3.org/2000/svg" width="1.877ex" height="2.021ex" role="img"
                            focusable="false" viewBox="0 -893.3 829.7 893.3" xmlns:xlink="http://www.w3.org/1999/xlink"
                            aria-hidden="true" style="vertical-align: 0px;">
                            <defs>
                                <path id="MJX-12-TEX-N-5B" d="M118 -250V750H255V710H158V-210H255V-250H118Z"></path>
                                <path id="MJX-12-TEX-N-31"
                                    d="M213 578L200 573Q186 568 160 563T102 556H83V602H102Q149 604 189 617T245 641T273 663Q275 666 285 666Q294 666 302 660V361L303 61Q310 54 315 52T339 48T401 46H427V0H416Q395 3 257 3Q121 3 100 0H88V46H114Q136 46 152 46T177 47T193 50T201 52T207 57T213 61V578Z">
                                </path>
                                <path id="MJX-12-TEX-N-5D" d="M22 710V750H159V-250H22V-210H119V710H22Z"></path>
                            </defs>
                            <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)">
                                <g data-mml-node="math">
                                    <g data-mml-node="msup">
                                        <g data-mml-node="mi"></g>
                                        <g data-mml-node="TeXAtom" transform="translate(33,363) scale(0.707)"
                                            data-mjx-texclass="ORD">
                                            <g data-mml-node="mo">
                                                <use data-c="5B" xlink:href="#MJX-12-TEX-N-5B"></use>
                                            </g>
                                            <g data-mml-node="mn" transform="translate(278,0)">
                                                <use data-c="31" xlink:href="#MJX-12-TEX-N-31"></use>
                                            </g>
                                            <g data-mml-node="mo" transform="translate(778,0)">
                                                <use data-c="5D" xlink:href="#MJX-12-TEX-N-5D"></use>
                                            </g>
                                        </g>
                                    </g>
                                </g>
                            </g>
                        </svg>
                        <mjx-assistive-mml unselectable="on" display="inline"><math
                                xmlns="http://www.w3.org/1998/Math/MathML">
                            </math></mjx-assistive-mml>
                    </mjx-container>
                    <script type="math/tex">^{[1]}</script><span> curve (</span><strong><span>Fig.
                            1</span></strong><span>) was
                        obtained, and the curve revealed the change of protein backbone stability in four temperature
                        gradients of
                        298.15 K, 313.15 K, 338.15 K and 400 K. Higher the RMSD value, poorer the stability of protein
                        backbone. As
                        shown in </span><strong><span>Fig. 1</span></strong><span>, with the increase of temperature,
                        the backbone
                        stability of MnP gradually decreased.</span>
                </p>
                <img src="./data/Result/DryLab/fig1.png" alt="">
                <p class="imgdescribe"><strong><span>Fig. 1 Average RMSD value of MnP at different
                            temperatures.</span></strong></p>
                <p>&nbsp;</p>
                <p><span> </span></p>
                <p><span>We also obtained the RMSF curve (</span><strong><span>Fig. 2</span></strong><span>) under four
                        reaction
                        temperatures of 288.15K, 313,15 K, 338.15 K, and 400 K through the analysis of the track file of
                        MD
                        simulation. The curve revealed that it was mainly position 228-233 residues of MnP that were
                        most sensitive to
                        temperature, thus facilitating the later construction of single mutation library. Meanwhile, we
                        compared the
                        RMSF</span>
                    <mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg
                            xmlns="http://www.w3.org/2000/svg" width="1.877ex" height="2.021ex" role="img"
                            focusable="false" viewBox="0 -893.3 829.7 893.3" xmlns:xlink="http://www.w3.org/1999/xlink"
                            aria-hidden="true" style="vertical-align: 0px;">
                            <defs>
                                <path id="MJX-13-TEX-N-5B" d="M118 -250V750H255V710H158V-210H255V-250H118Z"></path>
                                <path id="MJX-13-TEX-N-32"
                                    d="M109 429Q82 429 66 447T50 491Q50 562 103 614T235 666Q326 666 387 610T449 465Q449 422 429 383T381 315T301 241Q265 210 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H449V186Q448 183 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 137Q145 147 170 174T204 211T233 244T261 278T284 308T305 340T320 369T333 401T340 431T343 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 550Q109 549 114 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z">
                                </path>
                                <path id="MJX-13-TEX-N-5D" d="M22 710V750H159V-250H22V-210H119V710H22Z"></path>
                            </defs>
                            <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)">
                                <g data-mml-node="math">
                                    <g data-mml-node="msup">
                                        <g data-mml-node="mi"></g>
                                        <g data-mml-node="TeXAtom" transform="translate(33,363) scale(0.707)"
                                            data-mjx-texclass="ORD">
                                            <g data-mml-node="mo">
                                                <use data-c="5B" xlink:href="#MJX-13-TEX-N-5B"></use>
                                            </g>
                                            <g data-mml-node="mn" transform="translate(278,0)">
                                                <use data-c="32" xlink:href="#MJX-13-TEX-N-32"></use>
                                            </g>
                                            <g data-mml-node="mo" transform="translate(778,0)">
                                                <use data-c="5D" xlink:href="#MJX-13-TEX-N-5D"></use>
                                            </g>
                                        </g>
                                    </g>
                                </g>
                            </g>
                        </svg>
                        <mjx-assistive-mml unselectable="on" display="inline"><math
                                xmlns="http://www.w3.org/1998/Math/MathML">
                            </math></mjx-assistive-mml>
                    </mjx-container>
                    <script type="math/tex">^{[2]}</script><span> curve with the B-factor plot
                        (</span><strong><span>Fig.
                            3</span></strong><span>) from Bio3D, and the filtrated range of thermo-sensitive residues on
                        the two figures
                        was almost the same.</span>
                </p>
                <img src="./data/Result/DryLab/fig2.png" alt="">
                <p class="imgdescribe"><strong><span>Fig. 2 The RMS value of each residue of MnP at different
                            temperatures.</span></strong></p>
                <img src="./data/Result/DryLab/fig3.png" alt="">
                <p class="imgdescribe"><strong><span>Fig. 3 MnP B-Factor plot.</span></strong><em class="reference"><span>This curve reflects the
                    thermostability of each amino acid residue of the MnP.</span></em></p>
                    
                <h3>Secondary structure and salt bridge alterations</h3>
                <p><span>The secondary structure of MnP was also simulated and the results revealed through the
                        Ramachandran plot
                        and salt bridges.</span></p>
                <p><span>As shown in </span><strong><span>Fig. 4</span></strong><span>, the white area shown in the
                        Ramachandran
                        plot shrank with the increase of temperature, which represented the disruption of MnP’s
                        structure</span>
                    <mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg
                            xmlns="http://www.w3.org/2000/svg" width="1.877ex" height="2.021ex" role="img"
                            focusable="false" viewBox="0 -893.3 829.7 893.3" xmlns:xlink="http://www.w3.org/1999/xlink"
                            aria-hidden="true" style="vertical-align: 0px;">
                            <defs>
                                <path id="MJX-14-TEX-N-5B" d="M118 -250V750H255V710H158V-210H255V-250H118Z"></path>
                                <path id="MJX-14-TEX-N-33"
                                    d="M127 463Q100 463 85 480T69 524Q69 579 117 622T233 665Q268 665 277 664Q351 652 390 611T430 522Q430 470 396 421T302 350L299 348Q299 347 308 345T337 336T375 315Q457 262 457 175Q457 96 395 37T238 -22Q158 -22 100 21T42 130Q42 158 60 175T105 193Q133 193 151 175T169 130Q169 119 166 110T159 94T148 82T136 74T126 70T118 67L114 66Q165 21 238 21Q293 21 321 74Q338 107 338 175V195Q338 290 274 322Q259 328 213 329L171 330L168 332Q166 335 166 348Q166 366 174 366Q202 366 232 371Q266 376 294 413T322 525V533Q322 590 287 612Q265 626 240 626Q208 626 181 615T143 592T132 580H135Q138 579 143 578T153 573T165 566T175 555T183 540T186 520Q186 498 172 481T127 463Z">
                                </path>
                                <path id="MJX-14-TEX-N-5D" d="M22 710V750H159V-250H22V-210H119V710H22Z"></path>
                            </defs>
                            <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)">
                                <g data-mml-node="math">
                                    <g data-mml-node="msup">
                                        <g data-mml-node="mi"></g>
                                        <g data-mml-node="TeXAtom" transform="translate(33,363) scale(0.707)"
                                            data-mjx-texclass="ORD">
                                            <g data-mml-node="mo">
                                                <use data-c="5B" xlink:href="#MJX-14-TEX-N-5B"></use>
                                            </g>
                                            <g data-mml-node="mn" transform="translate(278,0)">
                                                <use data-c="33" xlink:href="#MJX-14-TEX-N-33"></use>
                                            </g>
                                            <g data-mml-node="mo" transform="translate(778,0)">
                                                <use data-c="5D" xlink:href="#MJX-14-TEX-N-5D"></use>
                                            </g>
                                        </g>
                                    </g>
                                </g>
                            </g>
                        </svg>
                        <mjx-assistive-mml unselectable="on" display="inline"><math
                                xmlns="http://www.w3.org/1998/Math/MathML">
                            </math></mjx-assistive-mml>
                    </mjx-container>
                    <script type="math/tex">^{[3]}</script><span>. </span>
                </p>
                <img src="./data/Result/DryLab/fig4.png" alt="">
                <p class="imgdescribe"><strong><span>Fig. 4 MnP Ramachandran plot at different
                            temperature.</span></strong><em><span class="reference"> This is a graphical
                            illustration for visualizing protein backbone energetic position in terms of torsion angles
                        </span></em>(
                            <mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg"
                                width="1.633ex" height="1.545ex" role="img" focusable="false" viewBox="0 -683 722 683"
                                xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: 0px;">
                                <defs>
                                  <path id="MJX-15-TEX-N-3A6"
                                    d="M312 622Q310 623 307 625T303 629T297 631T286 634T270 635T246 636T211 637H184V683H196Q220 680 361 680T526 683H538V637H511Q468 637 447 635T422 631T411 622V533L425 531Q525 519 595 466T665 342Q665 301 642 267T583 209T506 172T425 152L411 150V61Q417 55 421 53T447 48T511 46H538V0H526Q502 3 361 3T196 0H184V46H211Q231 46 245 46T270 47T286 48T297 51T303 54T307 57T312 61V150H310Q309 151 289 153T232 166T160 195Q149 201 136 210T103 238T69 284T56 342Q56 414 128 467T294 530Q309 532 310 533H312V622ZM170 342Q170 207 307 188H312V495H309Q301 495 282 491T231 469T186 423Q170 389 170 342ZM415 188Q487 199 519 236T551 342Q551 384 539 414T507 459T470 481T434 491T415 495H410V188H415Z">
                                  </path>
                                </defs>
                                <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)">
                                  <g data-mml-node="math">
                                    <g data-mml-node="mi">
                                      <use data-c="3A6" xlink:href="#MJX-15-TEX-N-3A6"></use>
                                    </g>
                                  </g>
                                </g>
                              </svg>
                            <script type="math/tex">\Phi</script><span> &amp; </span>
                            <mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg xmlns="http://www.w3.org/2000/svg"
                                width="1.76ex" height="1.545ex" role="img" focusable="false" viewBox="0 -683 778 683"
                                xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" style="vertical-align: 0px;">
                                <defs>
                                  <path id="MJX-16-TEX-N-3A8"
                                    d="M340 622Q338 623 335 625T331 629T325 631T314 634T298 635T274 636T239 637H212V683H224Q248 680 389 680T554 683H566V637H539Q479 637 464 635T439 622L438 407Q438 192 439 192Q443 193 449 195T474 207T507 232T536 276T557 344Q560 365 562 417T573 493Q587 536 620 544Q627 546 671 546H715L722 540V515Q714 509 708 509Q680 505 671 476T658 392T644 307Q599 177 451 153L438 151V106L439 61Q446 54 451 52T476 48T539 46H566V0H554Q530 3 389 3T224 0H212V46H239Q259 46 273 46T298 47T314 48T325 51T331 54T335 57T340 61V151Q126 178 117 406Q115 503 69 509Q55 509 55 526Q55 541 59 543T86 546H107H120Q150 546 161 543T184 528Q198 514 204 493Q212 472 213 420T226 316T272 230Q287 216 303 207T330 194L339 192Q340 192 340 407V622Z">
                                  </path>
                                </defs>
                                <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)">
                                  <g data-mml-node="math">
                                    <g data-mml-node="mi">
                                      <use data-c="3A8" xlink:href="#MJX-16-TEX-N-3A8"></use>
                                    </g>
                                  </g>
                                </g>
                              </svg>
                            </mjx-container>
                            <script type="math/tex">\Psi</script><span>).</span>
                          </p>
                <p><span>Moreover, its secondary structure (</span>
                    <mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg
                            xmlns="http://www.w3.org/2000/svg" width="1.281ex" height="2.034ex" role="img"
                            focusable="false" viewBox="0 -705 566 899" xmlns:xlink="http://www.w3.org/1999/xlink"
                            aria-hidden="true" style="vertical-align: -0.439ex;">
                            <defs>
                                <path id="MJX-17-TEX-I-1D6FD"
                                    d="M29 -194Q23 -188 23 -186Q23 -183 102 134T186 465Q208 533 243 584T309 658Q365 705 429 705H431Q493 705 533 667T573 570Q573 465 469 396L482 383Q533 332 533 252Q533 139 448 65T257 -10Q227 -10 203 -2T165 17T143 40T131 59T126 65L62 -188Q60 -194 42 -194H29ZM353 431Q392 431 427 419L432 422Q436 426 439 429T449 439T461 453T472 471T484 495T493 524T501 560Q503 569 503 593Q503 611 502 616Q487 667 426 667Q384 667 347 643T286 582T247 514T224 455Q219 439 186 308T152 168Q151 163 151 147Q151 99 173 68Q204 26 260 26Q302 26 349 51T425 137Q441 171 449 214T457 279Q457 337 422 372Q380 358 347 358H337Q258 358 258 389Q258 396 261 403Q275 431 353 431Z">
                                </path>
                            </defs>
                            <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)">
                                <g data-mml-node="math">
                                    <g data-mml-node="mi">
                                        <use data-c="1D6FD" xlink:href="#MJX-17-TEX-I-1D6FD"></use>
                                    </g>
                                </g>
                            </g>
                        </svg>
                        <mjx-assistive-mml unselectable="on" display="inline"><math
                                xmlns="http://www.w3.org/1998/Math/MathML">
                            </math></mjx-assistive-mml>
                    </mjx-container>
                    <script type="math/tex">\beta</script><span>-sheet part especially) was severely damaged
                        (</span><strong><span>Fig. 5</span></strong><span>), while the number of salt bridges increased
                        with the rise
                        of temperature which could be seen from the programming output. The above three indices
                        reflected the
                        disruption of protein structural stability caused by the increase of temperature.</span>
                </p>
                <img src="./data/Result/DryLab/fig5.png" alt="">
                <p class="imgdescribe"><strong><span>Fig. 5</span></strong><span> </span><strong><span>MnP
                            secondary structure changes with
                            temperature.</span></strong></p>
                <p><span> </span></p>
            </div>
            <div class="section" id="section2">
                <h2 id='optimization-of-mutant-library'><span>OPTIMIZATION OF MUTANT LIBRARY</span></h2>
                <h3>Rosetta</h3>
                <p><span>After building the library by FoldX</span>
                    <mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg
                            xmlns="http://www.w3.org/2000/svg" width="1.877ex" height="2.021ex" role="img"
                            focusable="false" viewBox="0 -893.3 829.7 893.3" xmlns:xlink="http://www.w3.org/1999/xlink"
                            aria-hidden="true" style="vertical-align: 0px;">
                            <defs>
                                <path id="MJX-18-TEX-N-5B" d="M118 -250V750H255V710H158V-210H255V-250H118Z"></path>
                                <path id="MJX-18-TEX-N-34"
                                    d="M462 0Q444 3 333 3Q217 3 199 0H190V46H221Q241 46 248 46T265 48T279 53T286 61Q287 63 287 115V165H28V211L179 442Q332 674 334 675Q336 677 355 677H373L379 671V211H471V165H379V114Q379 73 379 66T385 54Q393 47 442 46H471V0H462ZM293 211V545L74 212L183 211H293Z">
                                </path>
                                <path id="MJX-18-TEX-N-5D" d="M22 710V750H159V-250H22V-210H119V710H22Z"></path>
                            </defs>
                            <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)">
                                <g data-mml-node="math">
                                    <g data-mml-node="msup">
                                        <g data-mml-node="mi"></g>
                                        <g data-mml-node="TeXAtom" transform="translate(33,363) scale(0.707)"
                                            data-mjx-texclass="ORD">
                                            <g data-mml-node="mo">
                                                <use data-c="5B" xlink:href="#MJX-18-TEX-N-5B"></use>
                                            </g>
                                            <g data-mml-node="mn" transform="translate(278,0)">
                                                <use data-c="34" xlink:href="#MJX-18-TEX-N-34"></use>
                                            </g>
                                            <g data-mml-node="mo" transform="translate(778,0)">
                                                <use data-c="5D" xlink:href="#MJX-18-TEX-N-5D"></use>
                                            </g>
                                        </g>
                                    </g>
                                </g>
                            </g>
                        </svg>
                        <mjx-assistive-mml unselectable="on" display="inline"><math
                                xmlns="http://www.w3.org/1998/Math/MathML">
                            </math></mjx-assistive-mml>
                    </mjx-container>
                    <script type="math/tex">^{[4]}</script><span>, we made mutation free energy corrections for the
                        beneficial
                        mutation points that have been screened through Rosetta for a second database. Afterwards,
                        Funclib was used to
                        build a multi-functional enzyme library</span>
                    <mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg
                            xmlns="http://www.w3.org/2000/svg" width="1.877ex" height="2.021ex" role="img"
                            focusable="false" viewBox="0 -893.3 829.7 893.3" xmlns:xlink="http://www.w3.org/1999/xlink"
                            aria-hidden="true" style="vertical-align: 0px;">
                            <defs>
                                <path id="MJX-19-TEX-N-5B" d="M118 -250V750H255V710H158V-210H255V-250H118Z"></path>
                                <path id="MJX-19-TEX-N-35"
                                    d="M164 157Q164 133 148 117T109 101H102Q148 22 224 22Q294 22 326 82Q345 115 345 210Q345 313 318 349Q292 382 260 382H254Q176 382 136 314Q132 307 129 306T114 304Q97 304 95 310Q93 314 93 485V614Q93 664 98 664Q100 666 102 666Q103 666 123 658T178 642T253 634Q324 634 389 662Q397 666 402 666Q410 666 410 648V635Q328 538 205 538Q174 538 149 544L139 546V374Q158 388 169 396T205 412T256 420Q337 420 393 355T449 201Q449 109 385 44T229 -22Q148 -22 99 32T50 154Q50 178 61 192T84 210T107 214Q132 214 148 197T164 157Z">
                                </path>
                                <path id="MJX-19-TEX-N-5D" d="M22 710V750H159V-250H22V-210H119V710H22Z"></path>
                            </defs>
                            <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)">
                                <g data-mml-node="math">
                                    <g data-mml-node="msup">
                                        <g data-mml-node="mi"></g>
                                        <g data-mml-node="TeXAtom" transform="translate(33,363) scale(0.707)"
                                            data-mjx-texclass="ORD">
                                            <g data-mml-node="mo">
                                                <use data-c="5B" xlink:href="#MJX-19-TEX-N-5B"></use>
                                            </g>
                                            <g data-mml-node="mn" transform="translate(278,0)">
                                                <use data-c="35" xlink:href="#MJX-19-TEX-N-35"></use>
                                            </g>
                                            <g data-mml-node="mo" transform="translate(778,0)">
                                                <use data-c="5D" xlink:href="#MJX-19-TEX-N-5D"></use>
                                            </g>
                                        </g>
                                    </g>
                                </g>
                            </g>
                        </svg>
                        <mjx-assistive-mml unselectable="on" display="inline"><math
                                xmlns="http://www.w3.org/1998/Math/MathML">
                            </math></mjx-assistive-mml>
                    </mjx-container>
                    <script type="math/tex">^{[5]}</script><span> (</span><strong><span>Table. 1</span></strong><span>)
                        with
                        Rosetta</span>
                    <mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg
                            xmlns="http://www.w3.org/2000/svg" width="3.122ex" height="2.021ex" role="img"
                            focusable="false" viewBox="0 -893.3 1379.8 893.3" xmlns:xlink="http://www.w3.org/1999/xlink"
                            aria-hidden="true" style="vertical-align: 0px;">
                            <defs>
                                <path id="MJX-20-TEX-N-5B" d="M118 -250V750H255V710H158V-210H255V-250H118Z"></path>
                                <path id="MJX-20-TEX-N-36"
                                    d="M42 313Q42 476 123 571T303 666Q372 666 402 630T432 550Q432 525 418 510T379 495Q356 495 341 509T326 548Q326 592 373 601Q351 623 311 626Q240 626 194 566Q147 500 147 364L148 360Q153 366 156 373Q197 433 263 433H267Q313 433 348 414Q372 400 396 374T435 317Q456 268 456 210V192Q456 169 451 149Q440 90 387 34T253 -22Q225 -22 199 -14T143 16T92 75T56 172T42 313ZM257 397Q227 397 205 380T171 335T154 278T148 216Q148 133 160 97T198 39Q222 21 251 21Q302 21 329 59Q342 77 347 104T352 209Q352 289 347 316T329 361Q302 397 257 397Z">
                                </path>
                                <path id="MJX-20-TEX-N-2C"
                                    d="M78 35T78 60T94 103T137 121Q165 121 187 96T210 8Q210 -27 201 -60T180 -117T154 -158T130 -185T117 -194Q113 -194 104 -185T95 -172Q95 -168 106 -156T131 -126T157 -76T173 -3V9L172 8Q170 7 167 6T161 3T152 1T140 0Q113 0 96 17Z">
                                </path>
                                <path id="MJX-20-TEX-N-37"
                                    d="M55 458Q56 460 72 567L88 674Q88 676 108 676H128V672Q128 662 143 655T195 646T364 644H485V605L417 512Q408 500 387 472T360 435T339 403T319 367T305 330T292 284T284 230T278 162T275 80Q275 66 275 52T274 28V19Q270 2 255 -10T221 -22Q210 -22 200 -19T179 0T168 40Q168 198 265 368Q285 400 349 489L395 552H302Q128 552 119 546Q113 543 108 522T98 479L95 458V455H55V458Z">
                                </path>
                                <path id="MJX-20-TEX-N-5D" d="M22 710V750H159V-250H22V-210H119V710H22Z"></path>
                            </defs>
                            <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)">
                                <g data-mml-node="math">
                                    <g data-mml-node="msup">
                                        <g data-mml-node="mi"></g>
                                        <g data-mml-node="TeXAtom" transform="translate(33,363) scale(0.707)"
                                            data-mjx-texclass="ORD">
                                            <g data-mml-node="mo">
                                                <use data-c="5B" xlink:href="#MJX-20-TEX-N-5B"></use>
                                            </g>
                                            <g data-mml-node="mn" transform="translate(278,0)">
                                                <use data-c="36" xlink:href="#MJX-20-TEX-N-36"></use>
                                            </g>
                                            <g data-mml-node="mo" transform="translate(778,0)">
                                                <use data-c="2C" xlink:href="#MJX-20-TEX-N-2C"></use>
                                            </g>
                                            <g data-mml-node="mn" transform="translate(1056,0)">
                                                <use data-c="37" xlink:href="#MJX-20-TEX-N-37"></use>
                                            </g>
                                            <g data-mml-node="mo" transform="translate(1556,0)">
                                                <use data-c="5D" xlink:href="#MJX-20-TEX-N-5D"></use>
                                            </g>
                                        </g>
                                    </g>
                                </g>
                            </g>
                        </svg>
                        <mjx-assistive-mml unselectable="on" display="inline"><math
                                xmlns="http://www.w3.org/1998/Math/MathML">
                            </math></mjx-assistive-mml>
                    </mjx-container>
                    <script type="math/tex">^{[6,7]}</script><span>. Here we only considered the sites near the
                        substrate binding
                        pocket of the enzyme, and constrained the key active side chains and ion conformations, which
                        could narrow the
                        sequence sampling space to an exhaustive range. Through the combination of softrepack, repack,
                        rotmin and
                        other commonly used movers, we completed the scoring of the mutants. Later all the sites were
                        exhaustively
                        combined, and tolerable mutations were obtained as candidate mutation types. Finally, by sorting
                        and
                        clustering all the results, suggestions for mutants were obtained to expand the substrate
                        selectivity of
                        3M5Q.</span>
                </p>
                <p class="imgdescribe"><strong><span>Table. 1</span></strong><span> </span><strong><span>Calculated amino acid residue sets
                            at
                            available mutation sites under different ΔΔG values.</span></strong></p>
                <figure>
                    <table>
                        <thead>
                            <tr>
                                <th><span>Residue No. ΔΔG</span></th>
                                <th><span>35</span></th>
                                <th><span>39</span></th>
                                <th><span>177</span></th>
                            </tr>
                        </thead>
                        <tbody>
                            <tr>
                                <td><span>ΔΔG &lt;0.5</span></td>
                                <td><span>E</span></td>
                                <td><span>E</span></td>
                                <td><span>A, H, L, M, R, T, Y</span></td>
                            </tr>
                            <tr>
                                <td><span>0.5&lt;ΔΔG &lt;2.0</span></td>
                                <td><span>E</span></td>
                                <td><span>E</span></td>
                                <td><span>A, H, L, M, R, T, Y, K, S</span></td>
                            </tr>
                            <tr>
                                <td><span>2.0&lt;ΔΔG &lt;2.5</span></td>
                                <td><span>E, D</span></td>
                                <td><span>E</span></td>
                                <td><span>A, H, L, M, R, T, Y, Q</span></td>
                            </tr>
                            <tr>
                                <td><span>2.5&lt;ΔΔG &lt;3.0</span></td>
                                <td><span>E</span></td>
                                <td><span>E, D</span></td>
                                <td><span>A, H, L, M, R, T, Y</span></td>
                            </tr>
                        </tbody>
                    </table>
                </figure>
                <h3>DL &amp; ML analysis</h3>
                <p><span>We constructed the sequence library of the mutated enzyme, predicted</span>
                    <mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg
                            xmlns="http://www.w3.org/2000/svg" width="1.877ex" height="2.021ex" role="img"
                            focusable="false" viewBox="0 -893.3 829.7 893.3" xmlns:xlink="http://www.w3.org/1999/xlink"
                            aria-hidden="true" style="vertical-align: 0px;">
                            <defs>
                                <path id="MJX-21-TEX-N-5B" d="M118 -250V750H255V710H158V-210H255V-250H118Z"></path>
                                <path id="MJX-21-TEX-N-38"
                                    d="M70 417T70 494T124 618T248 666Q319 666 374 624T429 515Q429 485 418 459T392 417T361 389T335 371T324 363L338 354Q352 344 366 334T382 323Q457 264 457 174Q457 95 399 37T249 -22Q159 -22 101 29T43 155Q43 263 172 335L154 348Q133 361 127 368Q70 417 70 494ZM286 386L292 390Q298 394 301 396T311 403T323 413T334 425T345 438T355 454T364 471T369 491T371 513Q371 556 342 586T275 624Q268 625 242 625Q201 625 165 599T128 534Q128 511 141 492T167 463T217 431Q224 426 228 424L286 386ZM250 21Q308 21 350 55T392 137Q392 154 387 169T375 194T353 216T330 234T301 253T274 270Q260 279 244 289T218 306L210 311Q204 311 181 294T133 239T107 157Q107 98 150 60T250 21Z">
                                </path>
                                <path id="MJX-21-TEX-N-5D" d="M22 710V750H159V-250H22V-210H119V710H22Z"></path>
                            </defs>
                            <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)">
                                <g data-mml-node="math">
                                    <g data-mml-node="msup">
                                        <g data-mml-node="mi"></g>
                                        <g data-mml-node="TeXAtom" transform="translate(33,363) scale(0.707)"
                                            data-mjx-texclass="ORD">
                                            <g data-mml-node="mo">
                                                <use data-c="5B" xlink:href="#MJX-21-TEX-N-5B"></use>
                                            </g>
                                            <g data-mml-node="mn" transform="translate(278,0)">
                                                <use data-c="38" xlink:href="#MJX-21-TEX-N-38"></use>
                                            </g>
                                            <g data-mml-node="mo" transform="translate(778,0)">
                                                <use data-c="5D" xlink:href="#MJX-21-TEX-N-5D"></use>
                                            </g>
                                        </g>
                                    </g>
                                </g>
                            </g>
                        </svg>
                        <mjx-assistive-mml unselectable="on" display="inline"><math
                                xmlns="http://www.w3.org/1998/Math/MathML">
                            </math></mjx-assistive-mml>
                    </mjx-container>
                    <script type="math/tex">^{[8]}</script><span> the “scaled_effect” (</span><strong><span>Fig.
                            6</span></strong><span>) generated by the mutation with Turi Created applied, and merged it
                        into the final
                        .fasta file, which is used as the benchmark data set for deep learning (DL) training. According
                        to the weight
                        file generated after the calculation, support vector machines (SVM), random forest</span>
                    <mjx-container class="MathJax" jax="SVG" style="position: relative;"><svg
                            xmlns="http://www.w3.org/2000/svg" width="1.877ex" height="2.021ex" role="img"
                            focusable="false" viewBox="0 -893.3 829.7 893.3" xmlns:xlink="http://www.w3.org/1999/xlink"
                            aria-hidden="true" style="vertical-align: 0px;">
                            <defs>
                                <path id="MJX-22-TEX-N-5B" d="M118 -250V750H255V710H158V-210H255V-250H118Z"></path>
                                <path id="MJX-22-TEX-N-39"
                                    d="M352 287Q304 211 232 211Q154 211 104 270T44 396Q42 412 42 436V444Q42 537 111 606Q171 666 243 666Q245 666 249 666T257 665H261Q273 665 286 663T323 651T370 619T413 560Q456 472 456 334Q456 194 396 97Q361 41 312 10T208 -22Q147 -22 108 7T68 93T121 149Q143 149 158 135T173 96Q173 78 164 65T148 49T135 44L131 43Q131 41 138 37T164 27T206 22H212Q272 22 313 86Q352 142 352 280V287ZM244 248Q292 248 321 297T351 430Q351 508 343 542Q341 552 337 562T323 588T293 615T246 625Q208 625 181 598Q160 576 154 546T147 441Q147 358 152 329T172 282Q197 248 244 248Z">
                                </path>
                                <path id="MJX-22-TEX-N-5D" d="M22 710V750H159V-250H22V-210H119V710H22Z"></path>
                            </defs>
                            <g stroke="currentColor" fill="currentColor" stroke-width="0" transform="scale(1,-1)">
                                <g data-mml-node="math">
                                    <g data-mml-node="msup">
                                        <g data-mml-node="mi"></g>
                                        <g data-mml-node="TeXAtom" transform="translate(33,363) scale(0.707)"
                                            data-mjx-texclass="ORD">
                                            <g data-mml-node="mo">
                                                <use data-c="5B" xlink:href="#MJX-22-TEX-N-5B"></use>
                                            </g>
                                            <g data-mml-node="mn" transform="translate(278,0)">
                                                <use data-c="39" xlink:href="#MJX-22-TEX-N-39"></use>
                                            </g>
                                            <g data-mml-node="mo" transform="translate(778,0)">
                                                <use data-c="5D" xlink:href="#MJX-22-TEX-N-5D"></use>
                                            </g>
                                        </g>
                                    </g>
                                </g>
                            </g>
                        </svg>
                        <mjx-assistive-mml unselectable="on" display="inline"><math
                                xmlns="http://www.w3.org/1998/Math/MathML">
                            </math></mjx-assistive-mml>
                    </mjx-container>
                    <script type="math/tex">^{[9]}</script><span>, and K-nearest neighbor (KNN) were used to score the
                        model to
                        obtain the mutation activity of our enzyme, and the Spearman coefficients were accordingly 0.86,
                        0.88, and
                        0.92, which proved the result of “scaled_effect” after applying Turi Create was fine.</span>
                </p>
                <img src="./data/Result/DryLab/fig6.png" alt="">
                <p class="imgdescribe"><strong><span>Fig. 6 MnP scaled_effect.</span></strong><em class="reference"><span>The scatter points in the figure
                    are composed of the first principal component of PCA and the second principal component of
                    PCA. Variant
                    effect’s color reflects the confidence of scaled_effect.</span></em></p>
                <p><span>While such process was carrying on, we calculated the ectopic dominance of the protein after
                        mutation
                        through PLMC, and expressed the distance matrix (</span><strong><span>Fig.
                            7</span></strong><span>) between
                        its amino acids through the MATLAB code construction, which was helpful to better understand the
                        effect of
                        mutation on the surface of the protein.</span></p>
                <img src="./data/Result/DryLab/fig7.png" alt="">
                <p class="imgdescribe"><strong><span>Fig. 7 MnP PLMC module analysis and the distance matrix of
                            amino acid resideues.</span></strong>
                </p>
                <p><span> </span></p>
            </div>
            <div class="section" id="section3">
                <h2 id='cites'><span>REFERENCE</span></h2>
                <p class="reference"><span>[1] Berendsen, H. J., van der Spoel, D. &amp; van Drunen, R. GROMACS: a
                        message-passing parallel
                        molecular dynamics implementation. </span><em><span>Computer physics
                            communications</span></em><span>
                    </span><strong><span>91</span></strong><span>, 43-56 (1995).</span></p>
                <p class="reference"><span>[2] Sutherland, G. R. &amp; Aust, S. D. The effects of calcium on the thermal
                        stability and activity of
                        manganese peroxidase. </span><em><span>Archives of Biochemistry and Biophysics</span></em><span>
                    </span><strong><span>332</span></strong><span>, 128-134 (1996).</span></p>
                <p class="reference"><span>[3] Humphrey, W., Dalke, A. &amp; Schulten, K. VMD: visual molecular
                        dynamics. </span><em><span>Journal
                            of molecular graphics</span></em><span> </span><strong><span>14</span></strong><span>, 33-38
                        (1996).</span>
                </p>
                <p class="reference"><span>[4] Delgado, J., Radusky, L. G., Cianferoni, D. &amp; Serrano, L. FoldX 5.0:
                        working with RNA, small
                        molecules and a new graphical interface. </span><em><span>Bioinformatics</span></em><span>
                    </span><strong><span>35</span></strong><span>, 4168-4169 (2019).</span></p>
                <p class="reference"><span>[5] Ding, Y., Cui, K., Guo, Z., Cui, M. &amp; Chen, Y. Manganese peroxidase
                        mediated oxidation of
                        sulfamethoxazole: Integrating the computational analysis to reveal the reaction kinetics,
                        mechanistic
                        insights, and oxidation pathway. </span><em><span>Journal of Hazardous
                            Materials</span></em><span>
                    </span><strong><span>415</span></strong><span>, 125719 (2021).</span></p>
                <p class="reference"><span>[6] Fleishman, S. J. et al. RosettaScripts: a scripting language interface to
                        the Rosetta macromolecular
                        modeling suite. </span><em><span>PloS one</span></em><span>
                    </span><strong><span>6</span></strong><span>,
                        e20161 (2011).</span></p>
                <p class="reference"><span>[7] Rohl, C. A., Strauss, C. E., Misura, K. M. &amp; Baker, D. Protein
                        structure prediction using
                        Rosetta. </span><em><span>Methods in enzymology</span></em><span>
                    </span><strong><span>383</span></strong><span>, 66-93 (2004).</span></p>
                <p class="reference"><span>[8] Goldman, M. &amp; Pruitt, L. Comparison of the effects of gamma radiation
                        and low temperature
                        hydrogen peroxide gas plasma sterilization on the molecular structure, fatigue resistance, and
                        wear behavior
                        of UHMWPE. </span><em><span>Journal of biomedical materials research</span></em><span>
                    </span><strong><span>40</span></strong><span>, 378-384 (1998).</span></p>
                <p class="reference"><span>[9] Negi, S. S., Goldblum, R. M., Braun, W. &amp; Midoro-Horiuti, T. Design
                        of peptides with high
                        affinity binding to a monoclonal antibody as a basis for immunotherapy.
                    </span><em><span>Peptides</span></em><span> </span><strong><span>145</span></strong><span>, 170628
                        (2021).</span></p>
            </div>
        </div>
    </div>
    </div>
    <footer>
        <div id="sponsor">
            <img src="./data/Bottom/T--CPU_CHINA--Logoes.png" alt="">
        </div>
        <div id="connect">
            <h4>Contact us</h4>
            <ul id="contact">
                <li>
                    <a href="https://weibo.com/u/6054819039">
                        <img src="./data/Bottom/T--CPU_CHINA--weibologo.png" alt="">
                        <p>@CPU_CHINA2021</p>
                    </a>
                </li>
                <li>
                    <a href="mailto:cpu_china2021@163.com">
                        <img src="./data/Bottom/T--CPU_CHINA--emailLogo.png" alt="">
                        <p>cpu_china2021@163.com</p>
                    </a>
                </li>
                <li>
                    <a href="https://international.cpu.edu.cn/">
                        <img src="./data/Bottom/T--CPU_CHINA--CPUlogo.png" alt="">
                        <p>Interational Communication</p>
                    </a>
                </li>
            </ul>
        </div>
    </footer>
    <script
        src="https://2021.igem.org/wiki/index.php?title= Template:CPU_CHINA/VueJS &action=raw&amp;ctype=text/javascript">
        </script>
    <script
        src="https://2021.igem.org/wiki/index.php?title= Template:CPU_CHINA/Jquery &action=raw&amp;ctype=text/javascript">
        </script>
    <script src="./js/common.js"></script>
</body>

</html>